_extensions/odoo/translator.py

```diff
@@ -44,7 +45,16 @@ class BootstrapTranslator(nodes.NodeVisitor, object):
         '<meta name="viewport" content="width=device-width, initial-scale=1">'
     ]
 
-    def __init__(self, builder, document):
+    def __init__(self, arg1, arg2):
+        if int(sphinx.__version__.split('.')[0]) >= 2:
+            # TODO: remove this hack in V14.0
+            # This preserve compatibility between sphinx 1.6 and 2.3
+            # in python3, the arguments must be inverted.
+            builder = arg2
+            document = arg1
+        else:
+            builder = arg1
+            document = arg2
```

Comment:
Just check if the 1st param is the builder and swap if it's not, there's no reason to rely on version checks.